demos/interactive_face_detection_demo/face.cpp

```diff
@@ -81,17 +81,17 @@ size_t Face::getId() {
     return _id;
 }
 
-void Face::ageGenderEnable(bool value) {
-    _isAgeGenderEnabled = value;
+void Face::ageGenderEnable() {
+    _isAgeGenderEnabled = true;
 }
-void Face::emotionsEnable(bool value) {
-    _isEmotionsEnabled = value;
+void Face::emotionsEnable() {
+    _isEmotionsEnabled = true;
 }
-void Face::headPoseEnable(bool value) {
-    _isHeadPoseEnabled = value;
+void Face::headPoseEnable() {
+    _isHeadPoseEnabled = true;
 }
-void Face::landmarksEnable(bool value) {
-    _isLandmarksEnabled = value;
+void Face::landmarksEnable() {
+    _isLandmarksEnabled = true;
 }
```

Comment:
Do we need the above code? I assume all your decision making now happens in main

Reply:
`Face` is created with default fields `_isLandmarksEnabled` equal `false`
We need this code for correct visualization

Reply:
Did you do that?